bin/oneoff/script_clone_command_generator.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+# Generates the list of commands to clone each unit within an unit group
```

Comment:
I don't think this is the right location for this script, looking for other inputs

Reply:
I'm not sure about this. @davidsbailey, do you have any thoughts on where this should live?

Reply:
yes, this location is not quite right for scripts which will be reused. let's do `bin/curriculum/`

Reply:
Sounds good, I'll make that change in the last iteration so we can continue to do diffs between commits that address other comments.